src/styles/main.scss

```diff
@@ -21,7 +21,7 @@ main {
   min-height: calc(100vh - #{$header-height} - #{$footer-height});
 }
 
-.App-container {
+.app-container {
```

Comment:
This guy needs to stay as .App-container.

Reply:
Hmm, I don't remember touching that but I'll change it back.